pkgs/loom/default.nix

```diff
@@ -179,6 +180,15 @@ buildPythonPackage {
 
   passthru.loom-cpp = loom-cpp;
 
+  passthru.more_packages = {
+    inherit
+      goftests
+      distributions
+      pymetis
+      parsable
+    ;
+  };
+
```

Comment:
```suggestion

  passthru.ociImage = callPackage ./ociImage.nix { };
```

So you'd end up with `nix build .#loom.ociImage` rather than a separate sibling package.

Reply:
Interesting. I like the inheritance here.

Reply:
I am weighing this abstraction against like  `.#oci.loom` ...